src/objects/core/zcl_abapgit_file_status.clas.abap

```diff
@@ -9,6 +9,7 @@ CLASS zcl_abapgit_file_status DEFINITION
       IMPORTING
         !io_repo          TYPE REF TO zcl_abapgit_repo
         !ii_log           TYPE REF TO zif_abapgit_log OPTIONAL
+        !ii_pre_filter    TYPE REF TO zif_abapgit_repo_pre_filter OPTIONAL
```

Comment:
I expect no changes to the file status class, the repository can be filtered instead of passing it around